nimare/meta/cbma/ale.py

```diff
@@ -219,7 +236,31 @@ def just_histogram(*args, **kwargs):
         bin_edges = bin_centers - (step_size / 2)
         bin_edges = np.append(bin_centers, bin_centers[-1] + step_size)
 
-        ma_hists = np.apply_along_axis(just_histogram, 1, ma_values, bins=bin_edges, density=False)
+        if isinstance(ma_values, sparse._coo.core.COO):
+            masker = self.dataset.masker if not self.masker else self.masker
+            mask = masker.mask_img
+            mask_data = mask.get_fdata().astype(bool)
+            n_mask_voxels = np.count_nonzero(mask_data)
```

Comment:
I'm not sure how well this generalizes to other masker types, but I've opened an issue requesting a new attribute for masker objects with this info: https://github.com/nilearn/nilearn/issues/3296